dependencies/src/test/java/com/google/cloud/tools/opensource/classpath/LinkageCheckerMainIntegrationTest.java

```diff
@@ -136,7 +136,7 @@ public void testBom_java8()
       LinkageCheckerMain.main(new String[] {"-b", "com.google.cloud:libraries-bom:1.0.0"});
       fail("LinkageCheckerMain should throw LinkageCheckResultException upon errors");
     } catch (LinkageCheckResultException expected) {
-      assertEquals("Found 771 linkage errors", expected.getMessage());
+      assertEquals("Found 691 linkage errors", expected.getMessage());
```

Comment:
Adding default exclusion rules reduces the number of linkage errors reported in a BOM.